chat-plugins/economy.js

```diff
@@ -141,7 +142,7 @@ exports.commands = {
 		if (!target) target = user.name;
 
 		const amount = Db('money').get(toId(target), 0);
-		this.sendReplyBox(Tools.escapeHTML(target) + " has " + amount + currencyName(amount) + ".");
+		this.sendReplyBox("<b>font color='" + color(target) + "'>" + Tools.escapeHTML(target) + "</font></b> has " + amount + currencyName(amount) + ".");
```

Comment:
``"<b>font color='"``

You are missing the starting angle bracket ``<`` for the font tag.

``"<b><font color='"``